cli/cmd/inject.go

```diff
@@ -220,14 +224,31 @@ func InjectYAML(in io.Reader, out io.Writer, version string) error {
 		// objects, depending on the type.
 		var obj interface{}
 		var podTemplateSpec *v1.PodTemplateSpec
+		var DNSOverride string
 
+		/* When injecting the conduit proxy into a conduit controller pod. The conduit proxy's
+		   CONDUIT_PROXY_CONTROL_URL variable must be set to localhost for the following reasons:
+			1. According to https://github.com/kubernetes/minikube/issues/1568, minikube has an issue
+		       where pods are unable to connect to themselves through their associated service IP.
+		       Setting the CONDUIT_PROXY_CONTROL_URL to localhost allows the proxy to bypass kube DNS
+		       name resolution as a workaround to this issue.
+			2. We avoid the TLS overhead in encrypting and decrypting intra-pod traffic i.e. traffic
+		       between containers in the same pod.
+			3. Using a Service IP instead of localhost would mean intra-pod traffic would be load-balanced
+		       across all controller pod replicas. This is undesirable as we would want all traffic between
+			   containers to be self contained.
+			4. We skip recording telemetry for intra-pod traffic within the control plane.
+		*/
```

Comment:
Not sure why we would use `/* ... */` here instead of `//` comments here.

Reply:
I can convert it to `//` for consistency. It seemed easier to format when using `/* ...*/`. It looks like go prefers the latter style for package level comments.